README.md

```diff
@@ -4,9 +4,9 @@ The Trace Server Protocol is a protocol designed to allow frontend visualizers t
 
 * Keep things as imutable as possible
 * A trace can be queried for any UI elements.
-** UI elements can be but should not be limited to charts, graphs, trees and more.
-** Filtering on the server side may produce additional UI elements.
-** The data type is delivered to the client so the client can remain as lightweight as possible.
+ * UI elements can be but should not be limited to charts, graphs, trees and more.
+ * Filtering on the server side may produce additional UI elements.
+ * The data type is delivered to the client so the client can remain as lightweight as possible.
```

Comment:
The second commit should be squashed into the first/base one, here in this specific case.